src/hooks/useActionCenterSubscribers/useSwapActionSubscriber.tsx

```diff
@@ -132,10 +130,15 @@ export const useSwapActionSubscriber = () => {
         fetchIsSmartContractAddressQuery,
       })
 
+      const defaultExplorerBaseUrl =
+        buyTxHash && buyTxHash !== swap.sellTxHash
+          ? swap.buyAsset.explorerTxLink
+          : swap.sellAsset.explorerTxLink
+
       const txLink = getTxLink({
         address,
         chainId,
-        defaultExplorerBaseUrl: feeAsset?.explorerTxLink ?? '',
+        defaultExplorerBaseUrl,
```

Comment:
*not* introduced by this PR so let's not tackle for the time being but `getTxLink` it too optimistic and won't work when Tx isn't yet seen (swapper is affected by the same issue FWIW, it's not only a notifications thing)

i.e broadcast a THOR Tx on a slow conf chain like BTC, notice a dead viewblock link until seen and indexed by THOR network, and eventually by viewblock midgard indexer 

Probably worth following up on logic that checks on the granular status of the Tx (`getLatestThorTxStatusMessage` / `checkTradeStatus`) - basically, if the Tx status is anything but seen, we should use the L1 explorer link, not Viewblock.

Same for Butter and Relay FWIW (though Butter has had its Tx link removed so a non-issue).